docs/development/build-linux.md

```diff
@@ -1,85 +0,0 @@
-# Compiling LiveCode for Linux
```

Comment:
Is the removal of that file (`docs/development/build-linux.md`) part of this PR, or was it accidentally committed?